.gitignore

```diff
@@ -2,3 +2,6 @@ release/
 
 coverage.out
 drone-bintray
+
+*~
```

Comment:
This looks like editor specific ignores, you should add that to your local global gitignore.

Reply:
noted.